src/main.rs

```diff
@@ -7,12 +7,12 @@ extern crate log;
 use clap::{App, AppSettings, Arg, ArgMatches};
 use rustpython_compiler::compile;
 use rustpython_vm::{
-    import, match_class,
-    obj::{objint::PyInt, objtuple::PyTuple, objtype},
-    print_exception,
-    pyobject::{ItemProtocol, PyResult},
-    scope::Scope,
-    util, PySettings, VirtualMachine,
+	import, match_class,
```

Comment:
This formatting change doesn't look normal. Will you please update your rust by using `rustup` and run `cargo fmt --all` to format the code?

Reply:
I've change indent to 4 space